docs/content/preview/reference/configuration/operating-systems.md

```diff
@@ -23,7 +23,9 @@ Unless otherwise noted, operating systems are supported by all supported version
 | AlmaLinux 9      | {{<icon/yes>}} | {{<icon/yes>}} |       |
 | Oracle Linux 8   | {{<icon/yes>}} |                | |
 | Red Hat Enterprise Linux 8 | {{<icon/yes>}} |      | Recommended for production |
-| Red Hat Enterprise Linux&nbsp;9.3 and later| {{<icon/yes>}} |  | Supported in v2.20.3 and later.  {{<badge/ea>}} |
+| Red Hat Enterprise Linux 8 CIS Hardened | {{<icon/yes>}} |      | |
```

Comment:
any notes to add for Red Hat Enterprise Linux 8 CIS Hardened ? @yushenng . Is it  Recommended for production ?